src/acp/control.test.ts

```diff
@@ -228,3 +245,745 @@ describe("removeAccessControl", () => {
+    const control = createThing();
+
+    const policyUrls = getConstantMemberPolicyUrlAll(control);
+
+    expect(policyUrls).toHaveLength(0);
+  });
+});
+
+describe("removeConstantMemberPolicyUrl", () => {
+  it("removes the given Constant Member Policy URL from the Access Control", () => {
+    const policyUrl = "https://some.pod/policies#policy";
+    const control = addIri(createThing(), acp.applyMembersConstant, policyUrl);
+
+    const updatedControl = removeConstantMemberPolicyUrl(control, policyUrl);
+
+    const foundPolicyUrl = getIri(updatedControl, acp.applyMembersConstant);
+
+    expect(foundPolicyUrl).toBeNull();
+  });
+
+  it("does not remove other existing Constant Member Policies", () => {
+    const policyUrl = "https://some.pod/policies#policy";
+    const otherPolicyUrl = "https://some.pod/policies#other-policy";
+    let control = createThing();
+    control = addIri(control, acp.applyMembersConstant, policyUrl);
+    control = addIri(control, acp.applyMembersConstant, otherPolicyUrl);
+
+    const updatedControl = removeConstantMemberPolicyUrl(control, policyUrl);
+
+    expect(getIri(updatedControl, acp.applyMembersConstant)).toBe(
+      otherPolicyUrl
+    );
+  });
+
+  it("does not remove regular, Constant and Member Policy URLs", () => {
+    const policyUrl = "https://some.pod/policies#policy";
+    let control = createThing();
+    control = addIri(control, acp.apply, policyUrl);
+    control = addIri(control, acp.applyConstant, policyUrl);
+    control = addIri(control, acp.applyMembers, policyUrl);
+    control = addIri(control, acp.applyMembersConstant, policyUrl);
+
+    const updatedControl = removeConstantMemberPolicyUrl(control, policyUrl);
+
+    expect(getIri(updatedControl, acp.apply)).toBe(policyUrl);
+    expect(getIri(updatedControl, acp.applyConstant)).toBe(policyUrl);
+    expect(getIri(updatedControl, acp.applyMembers)).toBe(policyUrl);
+  });
+
+  it("accepts the URL to remove as a NamedNode", () => {
+    const policyUrl = "https://some.pod/policies#policy";
+    const control = addIri(createThing(), acp.applyMembersConstant, policyUrl);
+
+    const updatedControl = removeConstantMemberPolicyUrl(
+      control,
+      DataFactory.namedNode(policyUrl)
+    );
+
+    const foundPolicyUrl = getIri(updatedControl, acp.applyMembersConstant);
+
+    expect(foundPolicyUrl).toBeNull();
+  });
+
+  it("accepts the URL to remove as a ThingPersisted", () => {
+    const policyUrl = "https://some.pod/policies#policy";
+    const policy = createThing({ url: policyUrl });
+    const control = addIri(createThing(), acp.applyMembersConstant, policyUrl);
+
+    const updatedControl = removeConstantMemberPolicyUrl(control, policy);
+
+    const foundPolicyUrl = getIri(updatedControl, acp.applyMembersConstant);
+
+    expect(foundPolicyUrl).toBeNull();
+  });
+});
+
+describe("removeConstantMemberPolicyUrlAll", () => {
+  it("does not remove other existing Constant Member Policies", () => {
```

Comment:
```suggestion
  it("removes all Constant Member Policies", () => {
```

Reply:
How did that happen? And how on earth did you manage to spot that? :P 08e89b157e7a9fa607c2c18abaf7aa03a5ee2fd1